Makefile

```diff
@@ -17,11 +17,11 @@ build-image:
 
 test: test-unit test-e2e
 
-test-unit:
-	go test ./cmd/... ./pkg/...
+test-unit: verify
+	./hack/test-go.sh ./cmd/... ./pkg/...
```

Comment:
should we be consistent and set GOCACHE=off here too?

Reply:
GOCACHE=off is needed for tests that use external services. Our unit tests don't use network, so it's ok to cache them.